src/applications/personalization/profile-2/components/personal-information/PersonalInformation.jsx

```diff
@@ -3,35 +3,19 @@ import PropTypes from 'prop-types';
 import { Prompt } from 'react-router-dom';
 import { connect } from 'react-redux';
 
-import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
```

Comment:
moved the alert from this component up to the Profile2Wrapper component so it appears on all of the Profile pages.